src/style.css

```
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin-top: 20px;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.field {
+  box-sizing: border-box;
+
+  position: relative;
+  width: 100%;
+  height: 70px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  padding-left: 62px;
+  margin-bottom: 20px;
+}
+
+.field::placeholder {
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.field:focus::placeholder {
+  font-weight: bold;
+  line-height: 19px;
+}
+
+.field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.field:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+
+  outline: none;
+}
+
+.field--small {
+  height: 42px;
+  padding-left: 33px;
+}
+
+.field--small::placeholder {
+  font-size: 14px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.field--small:hover::placeholder {
```

```suggestion
.field--small:hover {
```
there is no sense in adding two pseudo classes like that